index.html

```diff
@@ -10469,16 +10469,30 @@ <h3>application/ld+json</h3>
             It is RECOMMENDED that profile URIs are dereferenceable and provide
             useful documentation at that URI. For more information and background
             please refer to [[RFC6906]].</p>
-          <p>This specification defines three values for the <code>profile</code> parameter.
-            To request or specify <a href="#expanded-document-form">expanded JSON-LD document form</a>,
-            the URI <code>http://www.w3.org/ns/json-ld#expanded</code> SHOULD be used.
-            To request or specify <a href="#compacted-document-form">compacted JSON-LD document form</a>,
-            the URI <code>http://www.w3.org/ns/json-ld#compacted</code> SHOULD be used.
-            To request or specify <a href="#flattened-document-form">flattened JSON-LD document form</a>,
-            the URI <code>http://www.w3.org/ns/json-ld#flattened</code> SHOULD be used.
-            Please note that, according [[HTTP11]], the value of the <code>profile</code>
-            parameter has to be enclosed in quotes (<code>"</code>) because it contains
-            special characters and, if multiple profiles are combined, whitespace.</p>
+          <p>This specification defines four values for the <code>profile</code> parameter.</p>
+          <dl>
+            <dt><code>http://www.w3.org/ns/json-ld#expanded</code></dt>
+            <dd>To request or specify <a href="#expanded-document-form">expanded JSON-LD document form</a>.</dd>
+            <dt><code>http://www.w3.org/ns/json-ld#compacted</code></dt>
+            <dd>To request or specify <a href="#compacted-document-form">compacted JSON-LD document form</a>.</dd>
+            <dt><code>http://www.w3.org/ns/json-ld#flattened</code></dt>
+            <dd>To request or specify <a href="#flattened-document-form">flattened JSON-LD document form</a>.</dd>
+            <dt><code>http://www.w3.org/ns/json-ld#framed</code></dt>
+            <dd>To request or specify <a href="#framed-document-form">framed JSON-LD document form</a>.</dd>
+          </dl>
+          <p>
+            When used as a <a data-cite="RFC2048#section-2.2.3">media type paramter</a> [[RFC2048]]
```

Comment:
I thought of that, as it reflects on the use of framing. But, I think it's a matter of interpretation if the use of the framing profile introduces new functionality, as the functionality is defined in the spec.

Reply:
Done in cd91bb9f0118a7a66029fcf963d3ee5bc0f4af4f.

Reply:
If "functionality" is specific to the mediatype (and that is my reading) then I would say that it's impossible for a _choice_ of frames to change the function, because the very function is... framing.